index.html

```diff
@@ -47,6 +47,157 @@ <h3>Let's CONNECT</h3>
+                        <li class="date">2015</li>
+                    </ul>
+                </div>
+                <div class="work-details">
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                </div>
+                <div class="work-skills">
+                    <ul>
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javascript</li>
+                    </ul>
+                </div>
+                <div class="work-button"><button>See Project</button></div>
+            </div>
+            <div class="work-container">
+                <div class="port-image"><img src="assets/work2.png" alt=""></div>
+                <div class="work-title"><h1>Multi-Post Stories</h1></div>
+                <div class="work-technology">
+                    <ul>
+                        <li class="">CANOPY</li>
+                        <li class=""><img src="assets/counter.png" alt=""></li>
+                        <li class="tech">Back End Dev</li>
+                        <li class=""><img src="assets/Counter.png" alt=""></li>
+                        <li class="date">2015</li>
+                    </ul>
+                </div>
+                <div class="work-details">
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                </div>
+                <div class="work-skills">
+                    <ul>
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javascript</li>
+                    </ul>
+                </div>
+                <div class="work-button"><button>See Project</button></div>
+            </div>
+            <div class="work-container">
+                <div class="port-image"><img src="assets/work3.png" alt=""></div>
+                <div class="work-title"><h1>Tonic</h1></div>
+                <div class="work-technology">
+                    <ul>
+                        <li class="">CANOPY</li>
+                        <li class=""><img src="assets/counter.png" alt=""></li>
+                        <li class="tech">Back End Dev</li>
+                        <li class=""><img src="assets/Counter.png" alt=""></li>
+                        <li class="date">2015</li>
+                    </ul>
+                </div>
+                <div class="work-details">
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                </div>
+                <div class="work-skills">
+                    <ul>
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javascript</li>
+                    </ul>
+                </div>
+                <div class="work-button"><button>See Project</button></div>
```

Comment:
- [ ] Kindly  follow the button styling for when it is  Active, Hover and Pressed check screenshot below ⬇️ 
![btn](https://user-images.githubusercontent.com/69638013/159018519-c77dc5dc-3b4c-475d-9344-ee807f96bddc.png)